client/src/store/search/actions.ts

```diff
@@ -42,14 +47,16 @@ export const getSearchResults = (query: string, page: number): (dispatch: Dispat
     async (dispatch: Dispatch): Promise<void> => {
         try {
             const FIRST_PAGE = 1;
+            const RESULTS_RER_PAGE = 5;
```

Comment:
Pls move to config and reuse

Reply:
I removed it from other place in code, so now I use it only once.